near-sdk/Cargo.toml

```
@@ -29,6 +29,9 @@ near-primitives-core = "=0.4.0"
 # Export dependencies for contracts
 wee_alloc = { version = "0.4.5", default-features = false, features = [] }
 
+# Used for caching, might be worth porting only functionality needed.
+once_cell = { version = "1.7.2", optional = true }
```

```suggestion
once_cell = { version = "1.7.2", optional = true, default-features = false }
```

I think need only no_std (single threaded ) version.